samples/Nancy.Demo.Razor.Localization/CustomResourceAssemblyProvider.cs

```diff
@@ -13,16 +11,22 @@
     /// </summary>
     public class CustomResourceAssemblyProvider : IResourceAssemblyProvider
     {
+        private readonly IAssemblyCatalog assemblyCatalog;
         private IEnumerable<Assembly> filteredAssemblies;
 
+        public CustomResourceAssemblyProvider(IAssemblyCatalog assemblyCatalog)
+        {
+            this.assemblyCatalog = assemblyCatalog;
+        }
+
         public IEnumerable<Assembly> GetAssembliesToScan()
         {
             return (this.filteredAssemblies ?? (this.filteredAssemblies = GetFilteredAssemblies()));
         }
 
-        private static IEnumerable<Assembly> GetFilteredAssemblies()
+        private IEnumerable<Assembly> GetFilteredAssemblies()
         {
-            return AppDomainAssemblyTypeScanner.Assemblies.Where(x => !x.IsDynamic);
+            return this.assemblyCatalog.GetAssemblies();

```

Comment:
Should just move the call to `GetAssemblies` to the end of line 24 and kill the method